README.md

```diff
@@ -7,35 +7,49 @@
 - Manage proxy admin rights.
 - Easily use in tests.
 
-## Quickstart
+## Installation
+
+### Buidler
 
-Install the plugin for your favorite toolchain:
 ```
-npm install --save-dev @openzeppelin/buidler-upgrades
+npm install --save-dev @openzeppelin/buidler-upgrades @nomiclabs/buidler-ethers ethers
```

Comment:
Rather than assuming that the user is already using Buidler, we should assume they *want* to use Buidler. Me suggestion then would be to include it in the installation command, same for Truffle.

Reply:
This is to guarantee successful tryouts, so I'd reconsider the `installation` title above in favor of something self contained, although I don't have strong feelings about it.

Reply:
> Rather than assuming that the user is already using Buidler, we should assume they want to use Buidler

I disagree with this. I see this section as "I'm using Buidler, I want to use this, what should I do".

The criteria about "guaranteeing successful tryouts" is interesting though. Let's keep the discussion going and keep iterating on this section.